modules/ROOT/pages/php.adoc

```diff
@@ -31,148 +31,15 @@ Whether you're using frameworks like Symfony and Laravel or you're developing wi
-
-[source,php]
-----
-<?php
-
-require_once 'vendor/autoload.php';
-
-use GraphAware\Neo4j\Client\ClientBuilder;
-
-$client = ClientBuilder::create()
-    ->addConnection('bolt', 'bolt://neo4j:password@localhost:7687')
-    ->build();
-$query = "MATCH (n:Person)-[:FOLLOWS]->(friend) RETURN n.name, collect(friend) as friends";
-$result = $client->run($query);
-
-foreach ($result->getRecords() as $record) {
-    echo sprintf('Person name is : %s and has %d number of friends', $record->value('name'), count($record->value('friends'));
-}
-----
-
-[cols="1,4"]
-|===
-| icon:user[] Author | http://twitter.com/ikwattro[Christophe Willemsen]
-| icon:play-circle[] Example | {examples}/movies-bolt-php
-| icon:gift[] Package | `composer require graphaware/neo4j-php-client:^4.0`
-| icon:github[] Source | https://github.com/graphaware/neo4j-php-client
-| icon:book[] Docs | https://github.com/graphaware/neo4j-php-client/blob/master/README.md
-|===
-
-[#graphaware-ogmphp]
-== GraphAware OGM for Neo4j and PHP
-
-GraphAware Neo4j-PHP-OGM is an object graph mapper for Neo4j in PHP. Highly inspired by Doctrine2, it uses annotations
-to defined mapping and all the operations to the graph are centralized via an EntityManager.
-
-[source,php]
-----
-<?php
-
-namespace Movies;
-
-use Doctrine\Common\Collections\ArrayCollection;
-use GraphAware\Neo4j\OGM\Annotations as OGM;
-
-/**
- * @OGM\Node(label="Movie")
- */
-class Movie
-{
-    /**
-     * @OGM\GraphId()
-     * @var int
-     */
-    protected $id;
-
-    /**
-     * @OGM\Property(type="string")
-     * @var string
-     */
-    protected $title;
-
-    /**
-     * @OGM\Property(type="string")
-     * @var string
-     */
-    protected $tagline;
-
-    /**
-     * @OGM\Property(type="int")
-     * @var int
-     */
-    protected $release;
-
-    /**
-     * @OGM\Relationship(type="ACTED_IN", direction="OUTGOING", targetEntity="Person", collection=true)
-     * @var ArrayCollection|Person[]
-     */
-    protected $actors;
-
-    /**
-     * @param string $title
-     * @param string|null $release
-     */
-    public function __construct($title, $release = null)
-    {
-        $this->title = $title;
-        $this->release = $release;
-        $this->actors = new ArrayCollection();
-    }
-    // Getters and Setters
-----
-
-[source, php]
-----
-<?php
+[#bolt]
+== Bolt
 
-$user = new User('cypher666');
-/** @var Movie $movie */
-$movie = $em->getRepository(Movie::class)->findOneBy('title', 'The Matrix');
-$user->rateMovie($movie, '4.5');
-$em->persist($user);
-$em->flush();
-----
+Bolt driver for PHP.
 
 [cols="1,4"]
 |===
-| icon:user[] Author | http://twitter.com/ikwattro[Christophe Willemsen]
-| icon:gift[] Package | `composer require graphaware/neo4j-php-ogm`
-| icon:github[] Source | https://github.com/graphaware/neo4j-php-ogm
-| icon:book[] Docs | https://github.com/graphaware/neo4j-php-ogm/blob/1.0/docs/01-intro.md
+| icon:user[] Author | https://www.linkedin.com/in/michalstefanak/[Michal Stefanak^]
```

Comment:
It would be nice to have a code example for this one too, like we used to have for the other drivers.

and a bit more docs/example/blog-post (can happen later)

Reply:
agreed, let me talk with Michal

Reply:
Let's include these when they happen.